lib/configParser.ts

```diff
@@ -0,0 +1,258 @@
+import { resolve, dirname } from 'path';
+import { sync } from 'glob';
+import { Logger } from './logger';
+
+// Coffee is required here to enable config files written in coffee-script.
+try {
+  require('coffee-script').register();
+} catch (e) {
+  // Intentionally blank - ignore if coffee-script is not available.
+}
+
+// LiveScript is required here to enable config files written in LiveScript.
+try {
+  require('LiveScript');
+} catch (e) {
+  // Intentionally blank - ignore if LiveScript is not available.
+}
+
+export interface Config {
+  specs: Array<string>;
+  multiCapabilities: Array<any>;
+  rootElement: string;
+  allScriptsTimeout: number;
+  getPageTimeout: number;
+  params: any;
```

Comment:
I am surprised that none of these are marked optional. Doesn't it fail to typecheck if you create a Config object that is missing one of the properties?